Sources/Containerization/Interface.swift

```diff
@@ -14,14 +14,16 @@
 // limitations under the License.
 //===----------------------------------------------------------------------===//
 
+import ContainerizationExtras
+
 /// A network interface.
 public protocol Interface: Sendable {
     /// The interface IPv4 address and subnet prefix length, as a CIDR address.
     /// Example: `192.168.64.3/24`
-    var address: String { get }
+    var address: CIDRv4 { get }
```

Comment:
nit (and we can do this in a separate change as well): I'm wondering if we should take the opportunity to also update these to be explicit

i.e. `ipv4Address` `ipv4Gateway`